Makefile

```diff
@@ -101,6 +101,10 @@ run:
 race:
 	find . -maxdepth 1 -type f -name '*.go' -not -name '*_test.go' | xargs go run -race -ldflags "-X main.program=$(PROGRAM) -X main.version=$(SVERSION)"
 
+# generate go files from non-go source
+bindata:
```

Comment:
Good, but we also need to change the build line to be:

`build: bindata $(PROGRAM)`